src/modules/FFmpeg/FormatContext.cpp

```diff
@@ -719,6 +719,13 @@ bool FormatContext::open(const QString &_url, const QString &param)
     // Useful, e.g. CUVID decoder needs valid PTS
     formatCtx->flags |= AVFMT_FLAG_GENPTS;
 
+    // enable sdp file & reduce udp latency
+    if (url.endsWith("sdp"))
+    {
+        av_dict_set(&options, "protocol_whitelist", "file,udp,rtp", 0);
```

Comment:
Is it possible to get whitelist and append rtp?

Reply:
I think it is safe to do so, if the `formatCtx` is only temporary to this playback session

Reply:
I just did a quick check, `formatCtx->codec_whitelist` is NULL before we call `formatCtx = openThr->getFormatCtx();`
so it seems no way to fetch and append the white list.

inside Ffmpeg source code avio.c:195 it seems to have a hard coded default_whitelist which is "file,crypto,data" for files if user supplies NULL

probably I can change it to "file,crypto,data,udp,rtp" in my commit?

Reply:
> file,crypto,data,udp,rtp

ok